packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -236,16 +236,16 @@ export function createTrackWire(property: keyof Tagging): Wire<Taggable> {
 }
 
 /**
- * Performs a track of a query with no results that used semantics as fallback.
- * The totalHits will be changed to -1 if semantic queries are found in order to differentiate
- * it from scenarios where the user encounters a no-results page without any semantic queries.
+ * Performs a track of a query with no results that used related prompts or semantic queries as fallback.
+ * The totalHits will be changed to -1 if related prompts or semantic queries are found in order to differentiate
+ * it from scenarios where the user encounters a no-results page without any related prompts or semantic queries.
  *
  * @public
  */
-export const trackNoResultsQueryWithSemanticsWire = filter(
+export const trackNoResultsQueryWithFallbackWire = filter(
```

Comment:
Hi @lauramargar, considering this is being exported, I wonder if this renaming should be treated as Breaking Change or not, what do you think?